### PR TITLE
add instructions on netlify function as event trigger

### PR DESCRIPTION
**Pull request by sw-yx**
### Description

Create Netlify event trigger instructions

### Affected components 

- Docs

### CODE REVIEW

**Comment by shahidhk** [comment]
Hi @sw-yx! Thanks for the PR! Can you sign the CLA so that I can get this merged? :smile:

**Comment by sw-yx** [comment]
signed

File: community/boilerplates/event-triggers/netlfy-functions/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Event trigger boilerplates on Zeit Now
```
Comment (tirumaraiselvan) [inline community/boilerplates/event-triggers/netlfy-functions/README.md]: Zeit Now :p

File: community/boilerplates/event-triggers/netlfy-functions/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Click on the following button to deploy GraphQL Engine on Heroku with the free Postgres add-on:
+
+[![Deploy to Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/hasura/graphql-engine-heroku)
+
+Checkout [docs](https://docs.hasura.io/1.0/graphql/manual/deployment/index.html) for other deployment options.
+
+
+###  Setup `netlify-cli`
+
+1. Create a Zeit account @ https://app.netlify.com
```
Comment (tirumaraiselvan) [inline community/boilerplates/event-triggers/netlfy-functions/README.md]: zeit account ?

File: community/boilerplates/event-triggers/netlfy-functions/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+   ntl init # or `ntl link` if you already have one and want to link the project
+   ```
+
+### Setup and deploy the trigger
+
+```
+ntl functions:create # and pick the hasura template
+```
+
+The code is located [here](https://github.com/netlify/netlify-dev-plugin/tree/master/src/functions-templates/js/hasura-event-triggered)
```
Comment (tirumaraiselvan) [inline community/boilerplates/event-triggers/netlfy-functions/README.md]: Should we document the process for creating a template?

Either that or we should put the source code here and document how to deploy that directly.

Comment (sw-yx) [reply community/boilerplates/event-triggers/netlfy-functions/README.md]: not needed, the process is literally `ntl functions:create`. and the deploy instructions is right there.

Comment (tirumaraiselvan) [reply community/boilerplates/event-triggers/netlfy-functions/README.md]: cool

Comment (sw-yx) [reply community/boilerplates/event-triggers/netlfy-functions/README.md]: thanks :)

**Pull request merged**